libs/utils/env.py

```diff
@@ -493,10 +476,16 @@ def install_tools(self, tools):
         :param tools: The list of names of tools to install
         :type tools: list(str)
         """
+
+        # Add tools dependencies
+        if 'rt-app' in tools:
+            tools += ['taskset', 'trace-cmd', 'perf', 'cgroup_run_into.sh',]
+
+        # Remove duplicates and already-instaled tools
+        tools = set(tools).difference(self.__installed_tools)
```

Comment:
The same here, by making tools a set first and then updating it with the difference.